crates/ln-dlc-node/src/node/dlc_channel.rs

```diff
@@ -167,7 +167,7 @@ impl<S: TenTenOneStorage + 'static, N: LnDlcStorage + Sync + Send + 'static> Nod
             .context("DLC channel to close not found")?;
 
         if force {
-            // TODO(bonomat): implement force closure
+            self.force_close_dlc_channel(&channel_id)?;
```

Comment:
❓ Should we add a test?

It might time to split up the new `ln-dlc-node` test into multiple ones.

Reply:
Done https://github.com/get10101/10101/pull/1806/commits/61e5ed46974a02facac7d0760a3364779741de8b